h/notification/reply.py

```diff
@@ -62,7 +62,7 @@ def get_notification(
     # Now we know we're dealing with a reply
     reply = annotation
 
-    parent = storage.fetch_annotation(request.db, parent_id)
+    parent = request.find_service(AnnotationReadService).get_annotation_by_id(parent_id)
```

Comment:
Oh why not just `annotation.parent`? because the whole references thing makes that not trivial :+1:

Reply:
No, I don't think there's any good reason really. I've swapped this over.